src/diffpy/utils/tools.py

```diff
@@ -206,3 +208,31 @@ def get_package_info(package_names, metadata=None):
         pkg_info.update({package: importlib.metadata.version(package)})
     metadata["package_info"] = pkg_info
     return metadata
+
+
+def compute_mu_using_xraydb(sample_composition, energy, density=None, packing_fraction=1):
+    """Compute the attenuation coefficient (mu) using the XrayDB database.
+
+    Computes mu based on the sample composition and energy.
+    User can provide a measured density or an estimated packing fraction.
+    Specifying the density is recommended, though not required for some pure or standard materials.
+    Reference: https://xraypy.github.io/XrayDB/python.html#xraydb.material_mu.
+
+    Parameters
+    ----------
+    sample_composition : str
+        The chemical formula or the name of the material.
+    energy : float
+        The energy in keV.
+    density : float, optional, Default is None
+        The mass density of the packed powder/sample in gr/cm^3.
```

Comment:
we need more info here about how these will be used, and what happens if someone specifies both.